packages/adapter-planetscale/src/planetscale.test.ts

```diff
@@ -1,10 +1,11 @@
 import { Client, connect } from '@planetscale/database'
+import { fetch as undiciFetch } from 'undici'
 
 import { PrismaPlanetScale } from './planetscale'
 
 describe('validation', () => {
   test('throws if passed Connection instance', () => {
-    const connection = connect({ url: 'http://example.com' })
+    const connection = connect({ url: 'http://example.com', fetch: undiciFetch })
```

Comment:
This change was somehow required for me to build & test locally.
It doesn't seem to hurt, though, so I'd keep it.

Reply:
Weird, this test should not get anywhere where `fetch` would've been required. But maybe planetscale changed something.

Reply:
I will try removing this and try again on the CI.

Reply:
https://github.com/prisma/prisma/pull/22953/commits/1a41661b0da4a6b8d0a10742af87c6b3dc91cfa4

Reply:
Looks like this was just a local problem of mine. I'm closing this conversation as the change has been reverted, and the CI is still green.